extensions/stripe-dashboard/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "stripe-dashboard",
+  "title": "Stripe Dashboard",
+  "description": "Quick Stripe Dashboard Access",
+  "icon": "stripe.png",
+  "author": "farisaziz12",
+  "categories": [
+    "Developer Tools",
+    "Productivity",
+    "Finance"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "View Dashboard",
+      "description": "Template for showing a static grid",
+      "mode": "view"
+    }
+  ],
+  "preferences": [
+    {
+      "name": "stripeLiveApiKey",
+      "type": "password",
+      "required": true,
+      "title": "Live Env API Key",
+      "description": "API Key for Live Environment. This can be found in your Stripe Dashboard under Developers > API Keys > Secret key",
+      "placeholder": "Enter your Stripe Live API Key"
+    },
+    {
+      "name": "stripeTestApiKey",
+      "type": "password",
+      "required": true,
```

Comment:
It is required because all requests in the test env will fail without. I can make this optional and add error handling when it's undefined and a test env request is being made.

Reply:
I have now made both the keys options to allow users to install the extension and only use the quick links. When using the commands that hit the Stripe API there is additional error handling to prompt the user to add the appropriate keys for the environment they want data from.